### PR TITLE
Add and fixing the SEO

### PR DESCRIPTION
**Pull request by patriciasky17**
Halo kak Sampang, ini aku ada ubah ubah dari project yang SEO, boleh kakak cek dulu sekiranya ada konten yang redundan atau gak terpakai? karena tadi kubandingin filenya dia ada beberapa berbeda, thank youu kak :D

### CODE REVIEW

File: che-calculator/mccabe-thiele-binary-distillation.html [hunk 1]
```diff
@@ -158,26 +159,31 @@ <h3>McCabe-Thiele Diagram</h3>
...
-            When \(\alpha &lt 1.05\), the separation process will take more trays as the components are difficult to seperate using this method. Relative volatility is as follows:
+            with vapor at certain temperature. This graph is used to calculate relative volatility (\(\alpha\)) to determine the amount of separation possible. Relative volatility
+            is as follows:
             \[\alpha= {{y_i/x_i}\over{y_j/x_j}}\]
             Where \(y\) is the vapor fraction, \(x\) is the liquid fraction, and \(i, j\) is the chemical components. In the binary system, the above equation can be expressed as
             \[\alpha= {{y_i/x_i}\over{(1-y_i)/(1-x_i)}}\]
             <br>
-            <h5>Operating line</h5>
-            Operating line is a simple line equation to determine amount of liquid per amount of vapour (\( \mathcal{L} / \mathcal{V}\)) in the distillation process.
+            For bubble point calculations which vapor-phase compositions are unknown, we can simplify the calculations by asumming \(\sum y_i=1\), so that the Raoult's law become
```
Comment (sampangeran) [inline che-calculator/mccabe-thiele-binary-distillation.html]: line 168 until 180 is obsolete. Please use the new one.

File: id/team.html [hunk 2]
```diff
@@ -139,47 +139,49 @@ <h2>Our Team</h2>
             <div class="row">
               <div class="col-12 text-center" data-aos="fade-down" >
                 <img src="./../assets/img/team-1.jpg" alt="img-team" class="img-fluid rounded-circle mb-3" style="width: 12.5rem;">
-                <p class="fw-bold mb-0">Co-Founder & Direktur Utama</p>
```
Comment (sampangeran) [inline id/team.html:142]: This file is obsolete. Please use the updated one

File: team.html [hunk 3]
```diff
@@ -147,12 +148,12 @@ <h2>Our Team</h2>
               <div class="col-md-6 col-lg-3 text-center" data-aos="fade-down-right">
                 <img src="./assets/img/team-2.jpg" alt="img-team" class="img-fluid rounded-circle mb-3" style="width: 12.5rem;">
                 <p class="fw-bold mb-0">Chief Operating Officer & Co-Founder</p>
-                <p>Felix Pratama, S.T., M.BA (Cand)</p>
```
Comment (sampangeran) [inline team.html:150]: The team's description is obsolete. Please use the new one

**Review by sampangeran — COMMENTED** [review]
some files are obsolete. Please update first to the new one

File: team.html (hunk 3 above)
Comment (patriciasky17) [reply team.html:150]: done, you can check it again :D

File: id/team.html (hunk 2 above)
Comment (patriciasky17) [reply id/team.html:142]: done, you can check it again :D

File: che-calculator/mccabe-thiele-binary-distillation.html (hunk 1 above)
Comment (patriciasky17) [reply che-calculator/mccabe-thiele-binary-distillation.html]: done, you can check it aagain :D

**Review by sampangeran — APPROVED** [review]

**Pull request merged**